### DISCUSSION TITLE
Login error

### DISCUSSION THREAD

**Discussion by AyaanJahanzebAhmed** [mitre/caldera#2878] [Q&A]
When  i try logging in ,I get a error  : "TypeError:n.response is undefined"

**Comment by mkultraWasHere** [comment]
Looking into now.

**Comment by mkultraWasHere** [comment]
@AyaanJahanzebAhmed Patched.

Either repull master with `--recursive` flag.

Or pull down latest from Magma plugin:
```
cd plugins/magma
git checkout master
git pull origin master
```

When starting Caldera server again, will need to pass `--build` flag for first booting up.

**Reply by mkultraWasHere** [reply]
Let us know if still having issues.

**Reply by activeshadow** [reply]
This worked for me. I can now login from Firefox. Thanks!

**Reply by mturnermrt** [reply]
I completed these steps, it got ride of the : "TypeError:n.response is undefined error however I am still haivng an issue, accounts and passwords are not working
